src/components/Tree/Tree.module.css

```diff
@@ -0,0 +1,62 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+.tree {
+    max-width: 800px;
```

Comment:
isn't there a var for that? or shouldn't the one who uses this component decide what the `max-width` is?

Reply:
there isn't a variable for that one, I'm fine letting the user decide what it should be

Reply:
removed the max-width and just wrapped it in storybook: acd5ea3